From cb4dabeaf4efbb1255ba9cd84efee12bc777dcf1 Mon Sep 17 00:00:00 2001
From: Keith Packard <keithp@keithp.com>
Date: Sun, 19 Oct 2008 13:39:04 -0700
Subject: [PATCH] Remove accidental 'return;' left at the top of I830DRIInitBuffers

I was testing the behaviour of the XAA-based DRI buffer drawing code for
tiled buffers and accidentally left I830DRIInitBuffers disabled.

Signed-off-by: Keith Packard <keithp@keithp.com>
---
 src/i830_dri.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/src/i830_dri.c b/src/i830_dri.c
index 944558b..ee37936 100644
--- a/src/i830_dri.c
+++ b/src/i830_dri.c
@@ -1203,7 +1203,6 @@ I830DRIInitBuffers(WindowPtr pWin, RegionPtr prgn, CARD32 index)
    int nbox;
    int buffer, first_buffer, last_buffer;
 
-   return;
    if (I810_DEBUG & DEBUG_VERBOSE_DRI)
       ErrorF("I830DRIInitBuffers\n");
 
-- 
1.6.0.3

